database/messages.js

```diff
@@ -1,69 +1,80 @@
 // connect to database
-require('./db');
+var db = require('./db');
 
 var mongoose = require('mongoose');
 var Promise = require('bluebird');
 mongoose.Promise = Promise;
 var messages = {};
 
 var MessageSchema = new mongoose.Schema({
+	id: Number,
```

Comment:
I started using `id` a few months ago because `_id` is not a good indexing property for an API...

it's confusing for developers:
```
> var f = db.shows.findOne()
> var i = f._id
> i
ObjectId("5877277847a8e651ff0e822f")
> i.str
5877277847a8e651ff0e822f
> db.shows.find({_id: i.str})
> // doesn't work
```

```
// will produce malformed json if included:
> printjson(f)
{..., "_id" : ObjectId("5877277847a8e651ff0e822f") }
```

and for users:
`uclaradio.com/shows/ObjectId("5877277847a8e651ff0e822f")` ?? what does that mean?

better to index:
`uclaradio.com/shows/23`